plugin/path_role_test.go

```diff
@@ -451,6 +464,520 @@ func TestRole_InvalidRoleType(t *testing.T) {
 	}, []string{"role type", invalidRoleType, "is invalid"})
 }
 
+func TestRetrieveRole(t *testing.T) {
+	type testCase struct {
+		name string
+
+		getName  string
+		getResp  *logical.StorageEntry
+		getErr   error
+		getTimes int
+
+		localMount            bool
+		localMountTimes       int
+		replicationState      consts.ReplicationState
+		replicationStateTimes int
+
+		putErr   error
```

Comment:
It looks like `putErr` is `nil` in all of the test cases, am I reading that correctly?

Reply:
Good catch. Removed. Storage errors were originally in the table tests but I pulled it out into its own test and neglected to remove it here.

Reply:
👍